### PR TITLE
add prefix path option

### PR DESCRIPTION
**Pull request by elmariofredo**
This way we can run local-npm in Docker container without any hassle

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ module.exports = function (argv) {
   var skimLocal = new PouchDB('skimdb', {
     auto_compaction: true
   });
-  var db = level('./binarydb');
+  var db = level(prefix + 'binarydb');
```
Comment (nolanlawson) [inline index.js]: Would be better to do path.resolve in case the user didn't put a /

**Comment by elmariofredo** [comment]
sorry I had it there not sure why I remove it :smile:

**Comment by jillesme** [comment]
:+1:

**Comment by nolanlawson** [comment]
Sweet, thanks a bunch! LGTM

**Comment by nolanlawson** [comment]
This needs a bit more work in order to get all the data actually isolated to the prefix. I'm working on it in another branch (`55-nolan`), but it also needs extensive testing because this repo (sadly) has no automated tests.

**Comment by nolanlawson** [comment]
Merged as c6ac68e033d4fcdbc0f49980fd3aa020088f791a and 2cbb7265ae6f22e421ccfed1f4a5b81757f2f63c; I also renamed it to `--directory`. Cheers!

**Comment by elmariofredo** [comment]
Thanks!

**Pull request closed without merging**